packages/seal/src/Reindex/PartialReindexConfig.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the CMS-IG SEAL project.
+ *
+ * (c) Alexander Schranz <alexander@sulu.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CmsIg\Seal\Reindex;
+
+final class PartialReindexConfig
+{
+    public function __construct(private \DateTimeInterface|\Closure $dateTimeBoundary, private \Generator|\Closure $identifiers)
```

Comment:
Can you give me some background information about the cases when this are closures or generators?